MySQL8: can't create run

### Description

Can't create a run when using MySQL8

### Steps to Reproduce

1. Try to create a run, using a library unit or new unit that you create via AT

**Expected behavior:**
A run should be created

**Actual behavior:**
An error is thrown on the server-side.
```
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'groups (name, parent_fk, OPTLOCK) values ('6', null, 0)' at line 1
```

### Additional Information
This is because 'groups' is a keyword starting in MySQL8.0.2.

--- a/src/main/java/org/wise/portal/domain/group/impl/PersistentGroup.java
+++ b/src/main/java/org/wise/portal/domain/group/impl/PersistentGroup.java
@@ -54,7 +54,7 @@ import org.wise.portal.domain.user.impl.UserImpl;
 public class PersistentGroup implements Group {
 
   @Transient
-  public static final String DATA_STORE_NAME = "groups";
+  public static final String DATA_STORE_NAME = "`groups`";
 
   @Transient
   public static final String USERS_JOIN_TABLE_NAME = "groups_related_to_users";

